sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/NumberVerificationSample.md

```diff
@@ -0,0 +1,31 @@
+# NumberVerification verify
+
+Number verification involves 2 steps. In the first request, you receive a redirect URL that must be followed, in order to get teh `ApcCode`. Then in the second request, the `ApcCode` gets sent in.
+
+```C#
+using Azure.Communication.ProgrammableConnectivity;
+
+private readonly Uri _endpoint = new Uri("https://eastus.prod.apcgatewayapi.azure.com");
```

Comment:
These samples need to be converted to snippets and should not have the have class members in them.

Reply:
Snippet has been added as a test with recordings, and no class members like `private` etc are in the snippet.